nginx/includes/redirects.conf

```diff
@@ -231,4 +231,4 @@ rewrite ^/docs/learn/encyclopedia/transactions-specialized/sponsored-reserves$ "
 rewrite ^/docs/build/guides/basics/create-account$ "/docs/build/guides/transactions/create-account" permanent;
 rewrite ^/docs/build/guides/basics/send-and-receive-payments$ "/docs/build/guides/transactions/send-and-receive-payments" permanent;
 
-
+rewrite ^/docs/learn/encyclopedia$ /docs/learn/fundamentals permanent;
```

Comment:
let's add another one that redirects `/docs/learn/encyclopedia/transactions-specialized`, just in case